test/test_main.py

```diff
@@ -34,27 +34,55 @@ def rule_disabled(code: str) -> bool:
     return next((exclude_disabled_rules(path) for path in feature_paths if os.path.basename(path).lower().startswith(code)), False)
 
 
+def contains_outcome_code(data, code='P00010'):
```

Comment:
Yes, I used it initially to parse the behave outcome in test_main.py, but decided to go for another sotution and forgot about this part. Deleted it. 
Instead of these tricks, it would probably be better to add the VAlidationOutcome to the JSON output as you suggest in a later comment.